web/service/Dockerfile

```diff
@@ -1,10 +1,19 @@
 FROM eclipse-temurin:17-alpine
 RUN addgroup -S spring && adduser -S spring -G spring
+
+RUN mkdir -p /newrelic/logs
+RUN chown -R spring:spring /newrelic/logs
+
 USER spring:spring
 ARG JAR_FILES=build/libs/\*.jar
 ARG PROPERTIES_FILE=build/resources/main/application.properties
 ARG CREDS_FILE=src/main/resources/web-based-gtfs-validator-a088ec5f045d.json
+
 COPY ${JAR_FILES} /
 COPY ${PROPERTIES_FILE} /
 COPY ${CREDS_FILE} /
-ENTRYPOINT ["java","-Xmx12g","-jar","/service-4.0.1-SNAPSHOT.jar"]
\ No newline at end of file
+
+ADD ./newrelic/newrelic.jar /newrelic/newrelic.jar
+ADD ./newrelic/newrelic.yml /newrelic/newrelic.yml
+
+ENTRYPOINT ["java","-Xmx12g","-javaagent:/newrelic/newrelic.jar","-jar","/service-4.0.1-SNAPSHOT.jar"]
```

Comment:
We should have the name of the jar as a docker file parameter. Otherwise, we will need to change the name of the file on each release. Another alternative is to have a `generic` name like `gtfs-validator-service.jar` and the builder of the docker image will guarantee the existence of the file with the appropriate name. We can address this outside of the scope of this PR.

Reply:
This has been addressed by #1398 🚀